### PR TITLE
Look and feel improvements (black buttons/labels to blue buttons/labels)

### PR DESCRIPTION
**Pull request by metiftikci**
I hate black buttons. Blue looks better for me. Is not it?
[Black 1](https://i.ibb.co/vQ7LNWf/black.jpg)
[Black 2](https://i.ibb.co/8xb4TZw/black2.jpg)
[Blue 1](https://i.ibb.co/PYx5mwB/blue.jpg)
[Blue 2](https://i.ibb.co/0csSD7t/blue2.jpg)

### CODE REVIEW

**Comment by metiftikci** [comment]
Ooh god! Sorry im noob on contributing. But this is also kind of UI ^_^
Should i do something @techknowlogick ?

Old
old

New
new

**Comment by lunny** [comment]
Please split this as two PRs.

**Review by sapk — APPROVED** [review]

**Comment by adelowo** [comment]
What does this look like on the dark theme?

File: templates/repo/commits_table.tmpl [hunk 1]
```diff
@@ -14,7 +14,7 @@
 						<input type="checkbox" name="all" id="all" value="true" {{.All}}>
 						<label for="all">{{.i18n.Tr "repo.commits.search_all"}} &nbsp;&nbsp;</label>
 					</div>
-					<button class="ui black tiny button" data-panel="#add-deploy-key-panel">{{.i18n.Tr "repo.commits.find"}}</button>
+					<button class="ui blue button" data-panel="#add-deploy-key-panel">{{.i18n.Tr "repo.commits.find"}}</button>
```
Comment (hexfs) [inline templates/repo/commits_table.tmpl:17]:
```suggestion
					<button class="ui blue tiny button" data-panel="#add-deploy-key-panel">{{.i18n.Tr "repo.commits.find"}}</button>
```

**Review by hexfs — CHANGES_REQUESTED** [review]
As @adelowo asked how does it look like on dark theme?

File: templates/repo/commits_table.tmpl (hunk 1 above)
Comment (metiftikci) [reply templates/repo/commits_table.tmpl:17]: It looks like better without 'tiny' class i guess

**Comment by ntimo** [comment]
To be honest, I think the black buttons look better. Why does this has to be changed?

**Review by kolaente — COMMENTED** [review]
Please resolve conflicts.

**Comment by muhammedtiftikci** [comment]
@silverwind can you make this change for me? It is hard to work on windows for me.
Im closing this PR, im sorry for that.

**Comment by silverwind** [comment]
Maybe yes. I'd probably do something closer to GitHub thought, e.g. a colored icon, not a "button".

**Pull request closed without merging**